examples/src/protocol_config.rs

```diff
@@ -0,0 +1,292 @@
+const STATUS_MSG_WASM_FILEPATH: &str = "./examples/res/status_message.wasm";
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    let worker = workspaces::sandbox().await?;
+    let wasm = std::fs::read(STATUS_MSG_WASM_FILEPATH)?;
+    let contract = worker.dev_deploy(&wasm).await?;
+
+    let outcome = contract
+        .call("set_status")
+        .args_json(serde_json::json!({
+            "message": "hello_world",
+        }))
+        .transact()
+        .await?;
+
+    let block_reference = {
+        let hash = outcome.outcome().block_hash;
+        near_primitives::types::BlockReference::BlockId(near_primitives::types::BlockId::Hash(
+            near_primitives::hash::CryptoHash(hash.0),
+        ))
+    };
+
+    // NOTE: this API is under the "experimental" flag and no guarantees are given.
+    let protocol_config = worker.protocol_config(block_reference).await?;
```

Comment:
Note, this will make the supplying of `block_reference` inconsistent with: https://github.com/near/workspaces-rs/blob/35151975c165b693ec795841107978ab5e6800b6/examples/src/various_queries.rs#L90

It would be great if these methods were utilized with the following struct https://github.com/near/workspaces-rs/blob/35151975c165b693ec795841107978ab5e6800b6/workspaces/src/rpc/query.rs#L57-L87 so that we can keep things consistent and not have the user do all boilerplate each time